graphql_compiler/schema_transformation/merge_schemas.py

```diff
@@ -764,22 +818,25 @@ def _scalars_match(scalar_name1, scalar_name2):
 
 
 def _add_edge_field(
-    source_type_node, sink_type_name, source_field_name, sink_field_name, edge_name, direction
-):
+    source_type_node: Union[InterfaceTypeDefinitionNode, ObjectTypeDefinitionNode],
+    sink_type_name: str,
+    source_field_name: str,
+    sink_field_name: str,
+    edge_name: str,
+    direction: str,
```

Comment:
Probably a good idea to use `typing.LITERAL['out', 'in']` here for the direction.

Reply:
This would be a nice to have if nothing prevents it from existing, but I'm happy to wait for a future PR as well.

Reply:
@pmantica1 discussed this a bit offline. Agreed, that it would be nice, but it is a bit trickier than adding the Literal type hint. We pass `OUT/INBOUND_EDGE_DIRECTION` to the function, which are constants with type `str`. This causes a mypy error. Possibly would be good to make `OUT/INBOUND_EDGE_DIRECTION` Literals as well, but seems outside the scope of this PR.

Reply:
Fair enough!